events/code_review_events/workflow.py

```diff
@@ -277,24 +273,26 @@ def __init__(self, cache_root):
             self.webserver.register(self.bus)
 
             # Create pulse listener
-            exchanges = []
+            exchanges = collections.defaultdict(list)
```

Comment:
If you remove the few lines from the bad merge below, you can probably just use a normal dict here.

Reply:
Done